### ISSUE TITLE
Past and future GDD optimum

### ISSUE THREAD

**Issue by vvandermeersch** [vvandermeersch/synchrony#1]
I used the paleoclimate data we generated for the article under review in ELE to compute the GDD optimum over the Holocene. Interestingly, it's quite stable: we could reference this supplementary figure when we say "_evolution towards a universal solstice trigger could make sense_".
![Sans titre](https://github.com/user-attachments/assets/91e77075-f86e-4f00-b7b7-2db0ab67de02)

I think it would be interesting if I do the same using the CMIP6 future projections I processed, to see where we might be headed.

**Comment by vvandermeersch** [comment]
And here's the result with future projections for 2080-2100 (5 GCMs):
![image](https://github.com/user-attachments/assets/ebd0075b-5a56-4b01-9b74-3880f9e13e1f)

_Edit: added SSP5_

**Comment by lizzieinvancouver** [comment]
@vvandermeersch Very cool! Have you also looked at a few sites to see if it is equally stable within sites (versus stable across sites because they flip around with one another)?

**Comment by vvandermeersch** [comment]
@lizzieinvancouver Good point! I took 10 random sites, and the stability seems to vary a lot between sites
![optimality_stability_sites](https://github.com/user-attachments/assets/34851de3-8d3c-43e8-a05e-e98cb9c427b9)

I guess I will need to run a more precise analysis to check the global trend?

**Comment by lizzieinvancouver** [comment]
@vvandermeersch Wow! That is super interesting. I think we'll also need to think about how this works if the variability within site is high across these big time periods. I think it would be good too to look at North America and see what the pattern looks like there.

**Comment by vvandermeersch** [comment]
@lizzieinvancouver I agree. It's less straightforward because I would need to generate the daily temperature variability first... but still possible!

Below, the stability of the optimal points (days for which _optimality_ is above the 95th percentile) across 150 sites:
![image](https://github.com/user-attachments/assets/2fad4ade-381f-4af0-ad2c-b94b2a4976fe)

And for 450 sites, across the four Holocene periods (here I used the **97.5th percentile** as a threshold to determine optimal days):
![image](https://github.com/user-attachments/assets/48bde150-2d31-4b00-acd2-08538a5eeab4)

**Comment by lizzieinvancouver** [comment]
@vvandermeersch Looks very cool! Do you mean these are sites with high optimality? I still need a little help on how to interpret these (what do the colors mean? Are they just organized by ID on the x axis? And maybe what you take away from these)....

**Comment by vvandermeersch** [comment]
@lizzieinvancouver True, all of this does lack a bit of explanation! Days considered as "optimum" are those where optimality (calculated over 30 years) is above the 95th percentile (or 97.5th). For each period (2000 BP, 5000 BP, ... shown in different colors---you’ll find the legend in one of the figures above), we can identify several potential optimum days, which are represented in the boxplot.

Here's an example for one site/one period:
![image](https://github.com/user-attachments/assets/2dac88ea-84d2-4c84-94a2-08927e499dbc)

**Comment by lizzieinvancouver** [comment]
@vvandermeersch Ah! Got it. But what is stability (it just looks like DOY)? And how is the x axis ordered? 

> Below, the stability of the optimal points (days for which _optimality_ is above the 95th percentile) across 150 sites:

**Comment by vvandermeersch** [comment]
@lizzieinvancouver  Indeed, the y-axis just represents DOY. Stability is rather the outcome of the figure, i.e. the more the values are spread in the boxplot the less stable optimum days are---within and across periods.
The x-axis is ordered by ID (related somehow to latitude and longitude).

**Comment by lizzieinvancouver** [comment]
@vvandermeersch Okay, that is what I thought but was not sure. So my take away would not that stable, but hard to know what is 'stable' without some null model or alternative model or such to help us.... We should think on this!

**Comment by vvandermeersch** [comment]
@lizzieinvancouver I agree. Seems stable on average (first figure of this issue), but when looking more closely we find important variation over space and time. This goes somewhat in the same direction as with historical data
